config/formats.ts

```diff
@@ -1867,11 +1874,12 @@ export const Formats: import('../sim/dex-formats').FormatList = [
 			}
 		},
 		onSwitchIn(pokemon) {
-			if (!pokemon.m.pokemove) return;
-			const pokemove = pokemon.m.pokemove;
-			if (!pokemove.exists) return;
-			// Place volatiles on the Pokémon to show the pokemove.
-			this.add('-start', pokemon, pokemove.name, '[silent]');
+			if (!pokemon.m.pokemoves?.length) return;
+			for (const pokemove of pokemon.m.pokemoves) {
+				if (pokemove.exists) {
+					this.add('-start', pokemon, pokemove.name, '[silent]');
+				}
+			}
```

Comment:
```suggestion
			for (const pokemove of pokemon.m.pokemoves) {
				this.add('-start', pokemon, pokemove.name, '[silent]');
			}
```
We already check if the pokemove exists in `onBegin`, so no need to do it again here